apps/web/pages/api/book/event.ts

```diff
@@ -623,7 +646,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   let booking: Booking | null = null;
   try {
     booking = await createBooking();
-    evt.uid = booking.uid;
+    evt.uid = booking?.uid ?? null;
```

Comment:
I got a doubt with this, in which scenarios our `evt.uid` can be `null`?

Reply:
@andreaestefania12  Do you happen to know when this happens? I wanna know a little more on this context

Reply:
Hi @alannnc, in case the function createBooking returns null, the booking.uid is going to be undefined but inside the interface CalendarEvent uid specifies that it should be a string or null, so that is the reason why I put null instead of undefined.
<img width="128" alt="image" src="https://user-images.githubusercontent.com/19562383/165090758-0aa647b5-473f-42b1-87b6-527a33d0ac99.png">